stories/Selector.story.tsx

```diff
@@ -1,4 +1,4 @@
-import React from 'react';
+import * as React from 'react';
```

Comment:
What's wrong with just doing `import React from React`?

Reply:
They look the same but they are different. 
`import React from 'react` imports the default export
`import * as React from 'react'` imports all named exports.

The second version is more accurate because the [typescript types](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/react/index.d.ts#L44) do not define a default export, but you don't really notice this in practice too much because many people enable [`synthetic default exports` ](https://www.typescriptlang.org/docs/handbook/compiler-options.html) which makese these imports seem interchangeable.

Reply:
I just put it so the typescript stops pointing the error... I think this is better because it will work properly regardless of your TS settings.